<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Button_start_analyze">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>150</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
		border-radius: 30px;
		background-color: rgb(156, 237, 255);
		font:  14pt &quot;微軟正黑體&quot;; 
}

QPushButton:hover{
        border:2px solid;
		border-color:  rgb(227, 227, 227);
        background-color:rgb(85, 219, 255);
}</string>
    </property>
    <property name="text">
     <string>開始分析</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_predict_CDR">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>421</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>病人預測結果 : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_select_video">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>150</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
		border-radius: 30px;
		background-color: rgb(156, 237, 255);
		font:  14pt &quot;微軟正黑體&quot;; 
}

QPushButton:hover{
        border:2px solid;
		border-color:  rgb(227, 227, 227);
        background-color:rgb(85, 219, 255);
}</string>
    </property>
    <property name="text">
     <string>選擇病人影片</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_origin_CDR">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>病人原先結果 : </string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
